README.md

```diff
@@ -605,6 +605,20 @@ Content of root's ~/.ssh/config.
 
 - *Default*: "# This file is being maintained by Puppet.\n# DO NOT EDIT\n"
 
+validate_sshd_config
+-----------------------
+Use Puppets >= 3.5 file type validate_cmd or stdlibs validate_cmd to test sshd_config for correctness.
+Note that the test on Puppet < 3.5 uses stdlibs validate_cmd function, which tests sshd_config during
+catalog compile time on the master. Valid values are 'true' and 'false'.
```

Comment:
We should use booleans instead of strings as the default. Please remove the quotes around true and false.